dist/gaia-header.js

```diff
@@ -191,7 +192,13 @@ exports.register = function(name, props) {
   Object.defineProperties(proto, descriptors);
 
   // Register the custom-element and return the constructor
-  return document.registerElement(name, { prototype: proto });
+  try {
+    return document.registerElement(name, { prototype: proto });
+  } catch (e) {
+    if (e.name !== 'NotSupportedError') {
+      throw e;
+    }
+  }
```

Comment:
Yeah Doug implemented it to catch errors thrown when the same element name is registered twice.